src/frontend/src/binder/expr/function.rs

```diff
@@ -1203,75 +1202,6 @@ impl Binder {
         }
     }
 
-    fn rewrite_format_to_concat_ws(inputs: Vec<ExprImpl>) -> Result<Vec<ExprImpl>> {
```

Comment:
I don't know is the change good or bad, because the old impl can report error earlier 😇

Reply:
Right. But it's not essential since we have to handle error at evaluation time anyways. It might be better to have an early check at binding time (e.g. if the format is constant, we ensure the number of arguments won't be too few.) Maybe we can add it back in the future. 😇